### PR TITLE
Remove string concatenation on logging statements

### PR DESCRIPTION
**Pull request by StrongestNumber9**

### CODE REVIEW

File: src/main/java/com/teragrep/rlo_13/MonitoredFileConsumer.java [hunk 1]
```diff
@@ -52,7 +52,7 @@ void readFile(MonitoredFile monitoredFile) {
         // !! trace log existing position
         FileChannel fileChannel = fileChannelCache.acquire(monitoredFile.getPath());
         if (fileChannel == null) {
-            LOGGER.trace("Gave up on <["+monitoredFile.getPath()+"]> due to null FileChannel.");
+            LOGGER.trace("Gave up on <[{}]> due to null FileChannel.", monitoredFile.getPath());
```
Comment (kortemik) [inline src/main/java/com/teragrep/rlo_13/MonitoredFileConsumer.java:55]: monitoredFile.getPath() is a method call, therefore must be guarded

File: src/main/java/com/teragrep/rlo_13/MonitoredFileConsumer.java [hunk 2]
```diff
@@ -62,7 +62,12 @@ void readFile(MonitoredFile monitoredFile) {
         try {
             long fileChannelSize = fileChannel.size();
             if (fileChannelSize < absolutePosition) {
-                LOGGER.trace("Path <["+monitoredFile.getPath()+"]> truncated: size: <"+fileChannelSize+"> is LT position <"+absolutePosition+">.");
+                LOGGER.trace(
+                        "Path <[{}]> truncated: size: <{}> is LT position <{}>.",
+                        monitoredFile.getPath(),
```
Comment (kortemik) [inline src/main/java/com/teragrep/rlo_13/MonitoredFileConsumer.java:67]: monitoredFile.getPath() is a method call, therefore must be guarded

File: src/main/java/com/teragrep/rlo_13/MonitoredFileConsumer.java [hunk 3]
```diff
@@ -104,7 +109,11 @@ void readFile(MonitoredFile monitoredFile) {
 
     @Override
     public void accept(MonitoredFile monitoredFile) {
-        LOGGER.trace("Accept path <["+monitoredFile.getPath()+"]> with status <"+monitoredFile.getStatus()+">");
+        LOGGER.trace(
+                "Accept path <[{}]> with status <{}>",
+                monitoredFile.getPath(),
+                monitoredFile.getStatus()
```
Comment (kortemik) [inline src/main/java/com/teragrep/rlo_13/MonitoredFileConsumer.java:115]: monitoredFile.getPath(),
                monitoredFile.getStatus()
are method calls, therefore needs guarding

**Review by kortemik — CHANGES_REQUESTED** [review]

**Comment by StrongestNumber9** [comment]
Implemented change requests

File: src/main/java/com/teragrep/rlo_13/FileChannelCache.java [hunk 4]
```diff
@@ -48,13 +48,13 @@ class FileChannelCache implements AutoCloseable {
...
         String cause = removalNotification.getCause().name();
-        LOGGER.trace("Entry removed because: " + cause);
+        LOGGER.trace("Entry removed because: {}", cause);
 
 
             try {
                 if (!activeFileChannels.containsKey(removalNotification.getKey())) {
                     // inactive, throw it out
-                    LOGGER.trace("Removal triggered for path <[" + removalNotification.getKey() + "]>");
+                    LOGGER.trace("Removal triggered for path <[{}]>", removalNotification.getKey());
```
Comment (kortemik) [inline src/main/java/com/teragrep/rlo_13/FileChannelCache.java:57]: method call in logging should be guarded

**Review by kortemik — CHANGES_REQUESTED** [review]

**Review by kortemik — APPROVED** [review]

**Pull request merged**